control-plane/subcommand/server-acl-init/command.go

```diff
@@ -538,7 +552,7 @@ func (c *Command) Run(args []string) int {
 	}
 
 	if c.flagCreateSnapshotAgentToken {
-		err := c.createLocalACL("client-snapshot-agent", snapshotAgentRules, consulDC, isPrimary, consulClient)
```

Comment:
The addition of `true` here signifies to `createACL()` whether or not to proceed with creating a kube secret and ACL token. If false it will stop before creating the ACL token.
As we slowly add support to other components we can back this change out.